client/utils/handle-logout.js

```diff
@@ -0,0 +1,24 @@
+import {toast} from "react-toastify";
+import {t} from "ttag";
+import {initialState} from "../reducers/organization";
+
+const redirectToStatus = (setUserData, userData, orgSlug) => {
+  setUserData({...userData, mustLogout: true});
+  window.location.assign(`${window.location.origin}/${orgSlug}/status`);
+};
+
+const handleLogout = (logout, cookies, orgSlug, setUserData, userData) => {
+  if (
+    userData.is_active === true &&
+    (userData.method === "" ||
+      (userData.method === "mobile_phone" && userData.is_verified === true) ||
+      (userData.method === "bank_card" && userData.is_verified === false))
+  ) {
+    redirectToStatus(setUserData, userData, orgSlug);
+  }
+  logout(cookies, orgSlug);
+  toast.error(t`ERR_OCCUR`);
```

Comment:
why is this an error? Can you add some comments please?
And if this is an error, does it means that if the if block above is triggered that's not an error? Shouldn't a return statement be present at the end of the if block above then?

Reply:
It's an error because either the token is wrong or user data is empty.